docs/breakpad_reporting.rst

```diff
@@ -163,20 +165,22 @@ development instance. There are a few options:
 
 4. On Linux, you can crash processes using ``kill``::
 
-       kill -ABRT $(pidof firefox)
+       kill -SIGFPE $(pidof firefox)
```

Comment:
`SIGFPE` doesn't work on MacOS for me, but `ABRT` does. You may want to note each for different platforms.